torchvision/models/detection/rpn.py

```diff
@@ -57,9 +57,7 @@ def __init__(
     ):
         super(AnchorGenerator, self).__init__()
 
-        if not isinstance(sizes[0], (list, tuple)):
```

Comment:
Here the previous implementation may be not robust enough. The 'if' statement wants to  make sure the elements in 'sizes' are tuples instead of int. I changed this to 'map' function just for robustness.